act/qc/__init__.py

```diff
@@ -19,15 +19,16 @@
         'bsrn_tests',
         'comparison_tests',
         'qcfilter',
-        'qctests',
         'radiometer_tests',
         'sp2',
     ],
     submod_attrs={
         'arm': ['add_dqr_to_qc'],
-        'qcfilter': ['QCFilter'],
-        'qctests': ['QCTests'],
```

Comment:
same here

Reply:
@mgrover1 see previous response.  If we are importing it directly at the start of the __init__ do we need it in the lazy_loader modules?